make.paws/R/make_package.R

```diff
@@ -98,7 +101,9 @@ copy_customizations <- function(api, path) {
   if (length(src_file) > 0) {
     package <- package_name(api)
     filename <- sprintf("%s_customizations.R", package)
-    file.copy(from = src_file, to = file.path(path, filename))
+    warning_text <- unlist(edit_warning(list(), new_line = T))
+    cat(warning_text, file = file.path(path, filename))
```

Comment:
Can you change this to `write` or `writeChar` instead of `cat`? I feel one of these would be clearer.

Reply:
I changed it to write